test/format/wts.c

```diff
@@ -336,6 +336,17 @@ configure_chunkcache(char **p, size_t max)
     }
 }
 
+/*
+ * configure_prefetch --
+ *     Configure prefetch settings for opening a connection.
+ */
+static void
+configure_prefetch(char **p, size_t max)
+{
+    if (GV(PREFETCH))
+        CONFIG_APPEND(*p, ",prefetch=(available=true,default=false)");
```

Comment:
Again, I suggest a small comment saying that we're allowing it for sessions or something like that.

Reply:
Done in 0b9acdf3832b511e5a6368abbd61fb982f25b58f